amr-wind/wind_energy/actuator/ActSrcLineOp.H

```diff
@@ -116,7 +116,17 @@ void ActSrcOp<ActTrait, ActSrcLine>::operator()(
             const auto pos_ip = wt * pos[ip] + (1.0 - wt) * opos[ip];
             const auto dist = cc - pos_ip;
             const auto dist_local = tmat[ip] & dist;
-            const auto gauss_fac = utils::gaussian3d(dist_local, eps[ip]);
+            // If the spanwise eps is zero use the two-dimensional Gaussian
+            // The order of epsilon is (chord, thickness, span)
+            amrex::Real gauss_fac;
+            if (eps[ip].z() < 1e-12) {
```

Comment:
is there a significance to 1e-12? If not, lets use std numeric limits for small numbers